build.gradle

```diff
@@ -1,53 +1,81 @@
-apply plugin: "java"
+plugins {
+    id 'java'
+    id 'maven-publish'
+}
 
-sourceCompatibility = '1.8'
-targetCompatibility = '1.8'
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+}
 
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    // jpf-core
-    implementation files(jpf.Jar.getFullPath())
+    // JPF-core dependency (adjust path as needed)
+    implementation files("/home/finn/jpf-push/jpf-core/build/jpf.jar")
```

Comment:
Please make sure the paths stay relative, as the absolute paths depends on the user name.